crates/bgp-pkt/src/wire/tests/path_attribute.rs

```diff
@@ -35,6 +36,9 @@ use netgauze_iana::address_family::{
 };
 use netgauze_parse_utils::{test_helpers::*, Span};
 
+use crate::iana::{BgpSidAttributeTypeError, IanaValueError};
+use crate::update::BgpUpdateMessage;
+use crate::wire::serializer::BgpMessageWritingError;
```

Comment:
can you apply `cargo fmt -- --config format_code_in_doc_comments=true --config wrap_comments=true --config imports_granularity=Crate` to cleanup the imports here